nucypher-core/src/reencryption.rs

```diff
@@ -182,3 +190,25 @@ impl<'a> ProtocolObjectInner<'a> for ReencryptionResponse {
 }
 
 impl<'a> ProtocolObject<'a> for ReencryptionResponse {}
+
+
+#[cfg(test)]
+mod tests {
+    use umbral_pre::encrypt;
+    use umbral_pre::SecretKey;
+    use crate::HRAC;
+
+    use super::ReencryptionRequest;
+
+    #[test]
+    fn conditions_and_context_are_different() {
+        let _some_secret = SecretKey::random();
+        let some_trinket = _some_secret.public_key();
```

Comment:
I take this as an undying and immutable endorsement of the trinket/secret nomenclature from @KPrasch.